### ISSUE TITLE
training log

### ISSUE THREAD

**Issue by kobe233333** [open-mmlab/mmpose#874]
If i can get value of AP or loss on train data when i do the training, just like the log information i found in [https://download.openmmlab.com/mmpose/bottom_up/higher_hrnet32_coco_512x512_20200713.log.json] ?
Therefore i can see if i get overfit result.

**Comment by kobe233333** [comment]
> The log will be saved in the work_dirs. Do you use a very small dataset?
> You may have to use a smaller log interval.
> https://github.com/open-mmlab/mmpose/blob/7d9b249958d72ad6fb7963cd210369c68db52309/configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/res50_coco_256x192.py#L23

but i can only see the ap score, no training loss

**Comment by jin-s13** [comment]
Please try `interval = 1`

**Comment by jin-s13** [comment]
Have you solved this problem?

**Comment by kobe233333** [comment]
> Have you solved this problem?

yes, appreciate!
